### PR TITLE
add configuration file for testing windows csi job

### PR DESCRIPTION
**Pull request by jingxu97**
This is a configuration file for testing windows csi job. Currently it
is a poc and test a personal repo

### CODE REVIEW

File: config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+presets:
+- labels:
+    preset-common-gce-windows: "true"
```
Comment (krzyzacy) [inline config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml:3]: this preset is shared across all the jobs, you don't have to redefine it: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/test-infra/17570/pull-test-infra-prow-checkconfig/1259913494262714368

Comment (jingxu97) [reply config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml]: got it. I removed it. Thank you!

File: config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+presubmits:
+  kubernetes-csi/csi-driver-windows-poc:
+  - name: pull-kubernetes-csi-csi-driver-windows-poc
+    always_run: true
```
Comment (krzyzacy) [inline config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml:4]: it probably should start with always_run: false

Comment (jingxu97) [reply config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml]: fixed. PTAL

File: config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      containers:
+      # We need this image because it has Docker in Docker and go.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200507-2ed1a50-master
+        command:
+        - runner.sh
+        args:
+        - bash
+        - -c
+        - >-
+          cd $(GOPATH)/src/k8s.io/kubernetes &&
```
Comment (BenTheElder) [inline config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml:31]: if you're going to CD to kubernetes in gopath you need to clone it using extra_refs

Comment (BenTheElder) [reply config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml:31]: or just skip this and use --extract=ci/latest

Comment (jingxu97) [reply config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml:31]: I am trying to run kubetest to start a windows cluster, so I assume I need to CD to kubernetes  or just use --extract=ci/latest without CD?

**Pull request merged**